client/src/components/Specification/Specification.jsx

```diff
@@ -16,11 +16,36 @@ const GETSPECIFICATION = gql`
 			requirements {
 				name
 				id
+				section {
+					name
+				}
 			}
 		}
 	}
 `;
 
+function sortRequirements(requirements) {
+	let sections = [];
+
+	for (let requirement of requirements) {
```

Comment:
I'd definitely think there's a less imperative way to do this. Isn't there a built in array sort? Edit - it's not actually being sorted is it? Can you not just use map?